extensions/standard-processors/processors/HashContent.h

```diff
@@ -72,7 +64,7 @@ namespace {
     if (ret_val.second > 0) {
       unsigned char digest[MD5_DIGEST_LENGTH];
       MD5_Final(digest, &context);
-      ret_val.first = digestToString(digest, MD5_DIGEST_LENGTH);
+      ret_val.first = utils::StringUtils::to_hex(digest, MD5_DIGEST_LENGTH, true /*uppercase*/);
```

Comment:
ditto here, too. If you wish submit the to_hex pr changing variants let's get that as a separate PR so we can accurately prioritize its review. Thanks!